internal/driver/html/common.js

```diff
@@ -563,11 +563,11 @@ function viewer(baseUrl, nodes, options) {
     return str.replace(/([\\\.?+*\[\](){}|^$])/g, '\\$1');
   }
 
-  function setSampleIndexLink(id) {
-    const elem = document.getElementById(id);
+  function setSampleIndexLink(si) {
+    const elem = document.getElementById('sampletype-'+si);
```

Comment:
nit: I think the grammar convention in this file is having space before/after the operator +. Can you change it to that?

Reply:
Done. Thanks.